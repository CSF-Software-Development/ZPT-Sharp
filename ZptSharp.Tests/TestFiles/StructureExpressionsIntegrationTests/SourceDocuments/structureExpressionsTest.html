<html>
    <head>
        <title>Structure expressions test</title>
    </head>
    <body tal:define="isNotStructure here/myMarkup;
                      isStructure structure:here/myMarkup">
        <h1>Structure expression test</h1>
        <div tal:content="isNotStructure">This will be filled with encoded text.</div>
        <div tal:content="isStructure">This will be filled with HTML.</div>
    </body>
</html>