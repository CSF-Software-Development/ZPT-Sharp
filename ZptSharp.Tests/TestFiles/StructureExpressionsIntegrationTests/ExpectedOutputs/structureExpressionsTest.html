<html>
    <head>
        <title>Structure expressions test</title>
    </head>
    <body>
        <h1>Structure expression test</h1>
        <div>&lt;p&gt;Hello world!&lt;/p&gt;</div>
        <div><p>Hello world!</p></div>
    </body>
</html>